packages/react-magma-dom/src/components/TreeView/useTreeView.ts

```diff
@@ -57,6 +57,17 @@ export interface UseTreeViewProps {
    * Example: [ {itemId: 'item0', checkedStatus: IndeterminateCheckboxStatus.indeterminate}, {itemId: 'item0-child', checkedStatus: IndeterminateCheckboxStatus.checked} ]
    */
   onSelectedItemChange?: (selectedItems: Array<Object>) => void;
+  /**
+   * Determines if the parent checkbox will get selected when the user selects all its children checkboxes.
```

Comment:
Can we note that these two props only affects `TreeViewSelectable.multi`?

Reply:
note added